config/rbac-rotation/role.yaml

```diff
@@ -0,0 +1,16 @@
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Creating a separate file similar to how we do it today for secret sync rbac roles